mesonbuild/dependencies/base.py

```diff
@@ -1449,24 +1628,25 @@ def _setup_compiler(self, build_dir):
 set(CMAKE_SIZEOF_VOID_P "{}")
 '''.format(os.path.realpath(__file__), ctypes.sizeof(ctypes.c_voidp)))
 
-    def _setup_cmake_dir(self):
+    def _setup_cmake_dir(self, cmake_file: str):
```

Comment:
shouldn't this also have `-> str` at the end?

Reply:
Yes, it should. Updated.